### ISSUE TITLE
multilanguage preview not working?

### ISSUE THREAD

**Issue by ndac-todoroki** [FontManager/font-manager#70]
I'm on `0.7.3`.

My system language is Japanese, and all fonts tries to preview in a Japanese example sentence. (not _"A quick brown fox..."_)
I thought this was supposed to be fixed by #16, but somehow it is not working.

Example captures: `Ubuntu` and `Noto Sans CJK`
![Ubuntu font example](https://user-images.githubusercontent.com/11530165/51518311-d32ac500-1e60-11e9-9120-b65bde54a553.png)
![Noto Sans CJK example](https://user-images.githubusercontent.com/11530165/51518327-d920a600-1e60-11e9-9464-8dacba26e14d.png)

**Comment by JerryCasiano** [comment]
Ubuntu font doesn't appear to support Japanese so...

Probably a lot of fonts display this way on your system and I can see how that would be very annoying.

Should we just fallback to Basic Latin and try to display "A quick brown fox..."?

**Comment by ndac-todoroki** [comment]
Thanks for the information. 
I wonder how Windows and Gnome Fonts handle this. It shows Japanese for Japanese fonts, and English for Latin fonts.

> Should we just fallback to Basic Latin and try to display "A quick brown fox..." in that case?

Yes, maybe this is the solution. Some example sentence is set per language (as translations?), and if the characters aren't there, it will fallback to Basic Latin.

BTW, #13 can be one of the workaround for this.
For example in Japanese we use both CJK letters and Latin letters in the same sentence, so it'd be great if I could set my original sample including both of them.

**Comment by JerryCasiano** [comment]
I'll have a look at how gnome-fonts handles this.

I thought they did the same thing we do where if the font doesn't support Latin it renders a random sample with whatever is found in the font.

Also, just FYI none of the issues closed recently such as #16 apply to 0.7.3 only to the latest release, not that it fixes your issue anyway but there are a LOT of changes in 0.7.4.

In any case, this is an area that definitely needs improvement. 

Thanks for bringing this up.

**Comment by JerryCasiano** [comment]
I believe the changes in master should resolve or at the very least improve this situation.

Please test and provide feedback.

**Comment by clee231** [comment]
Not the original poster, but I had a similar issue with English fonts and any other non-latin or emoji fonts.
This issue was resolved when I built and ran the latest version from master (0.7.4 @ 109c734).
No more box fonts is great! Thanks for the fix!

Screenshots:
![2019-02-14-215255_1390x1110_scrot](https://user-images.githubusercontent.com/1387910/52833617-4069f880-30a3-11e9-9b2c-cb5425d65cbf.png)
![2019-02-14-215108_1390x1110_scrot](https://user-images.githubusercontent.com/1387910/52833625-452eac80-30a3-11e9-8532-f457fbf7dcaf.png)
